### PR TITLE
Edit some parts of documentation & respect PEP8 conventions

### PR DESCRIPTION
**Pull request by hearot**

### CODE REVIEW

File: backend/pycon/settings/base.py [hunk 1]
```diff
@@ -74,16 +74,20 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+        'NAME':
```
Comment (simobasso) [inline backend/pycon/settings/base.py]: I prefer this in one line

Comment (simobasso) [reply backend/pycon/settings/base.py]: (and the others below :) )

File: backend/users/admin.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 
 class PyConUserCreationForm(UserCreationForm):
+
```
Comment (simobasso) [inline backend/users/admin.py]: I prefer no blank lines before class docstring

File: backend/users/admin.py [hunk 3]
```diff
@@ -18,6 +19,7 @@ class Meta:
 
 
 class PyConUserChangeForm(UserChangeForm):
+
```
Comment (simobasso) [inline backend/users/admin.py]: same as above 😄

File: backend/users/migrations/0001_initial.py [hunk 4]
```diff
@@ -18,15 +18,24 @@ class Migration(migrations.Migration):
...
-                ('last_login', models.DateTimeField(blank=True, null=True, verbose_name='last login')),
-                ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
-                ('email', models.EmailField(max_length=254, unique=True, verbose_name='email address')),
-                ('date_joined', models.DateTimeField(auto_now_add=True, verbose_name='date joined')),
-                ('is_active', models.BooleanField(default=True, verbose_name='active')),
-                ('groups', models.ManyToManyField(blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of their groups.', related_name='user_set', related_query_name='user', to='auth.Group', verbose_name='groups')),
-                ('user_permissions', models.ManyToManyField(blank=True, help_text='Specific permissions for this user.', related_name='user_set', related_query_name='user', to='auth.Permission', verbose_name='user permissions')),
+                ('id', models.AutoField(auto_created=True,
```
Comment (simobasso) [inline backend/users/migrations/0001_initial.py]: please revert this one, change a migration already applied isn't a good idea 😆 (we must exclude migrations from flake8/pep8/whatever)

File: backend/users/migrations/0003_user_username.py [hunk 5]
```diff
@@ -15,6 +15,10 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='user',
             name='username',
-            field=models.CharField(blank=True, max_length=100, null=True, verbose_name='username'),
+            field=models.CharField(
```
Comment (simobasso) [inline backend/users/migrations/0003_user_username.py]: please revert this one, change a migration already applied isn't a good idea 😆 (we must exclude migrations from flake8/pep8/whatever)

File: backend/users/models.py [hunk 6]
```diff
@@ -8,7 +8,11 @@
 
 
 class User(AbstractBaseUser, PermissionsMixin):
-    username = models.CharField(_('username'), max_length=100, null=True, blank=True)
+    username = models.CharField(
+        _('username'),
+        max_length=100,
+     null=True,
+     blank=True)
```
Comment (simobasso) [inline backend/users/models.py]: 8 spaces instead of 4 on LOC 14 and 15, and the last parentheses a new line

**Comment by simobasso** [comment]
Hi @hearot thanks for your pr!

I've just reviewed the pull request and I've to ask you few changes. Feel free to ping me if you need!

😄

**Comment by hearot** [comment]
@simobasso Just edited what you said, tell me if all is right.

**Comment by simobasso** [comment]
closed in favour of #154

**Pull request closed without merging**